crates/system-reinstall-bootc/Cargo.toml

```diff
@@ -21,16 +21,16 @@ clap = { workspace = true, features = ["derive"] }
 crossterm = "0.29.0"
 dialoguer = "0.11.0"
 indoc = { workspace = true }
-log = "0.4.21"
+log = "0.4.27"
 openssh-keys = "0.6.4"
 rustix = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
 serde_json = { workspace = true }
-serde_yaml = "0.9.22"
+serde_yaml = "0.9.34"
```

Comment:
I think we should actually be moving more of these common dependencies into the workspace - that's a different PR

Reply:
:arrow_right: https://github.com/bootc-dev/bootc/pull/1501